### PR TITLE
Feat: LDIO component debug logger

### PR DESCRIPTION
**Pull request by Yalz**
To be merged after sprint change.

### CODE REVIEW

File: ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/AdapterDebugger.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	private final LdiAdapter adapter;
+
+	public AdapterDebugger(LdiAdapter adapter) {
+		this.logger = LoggerFactory.getLogger(adapter.getClass());
+		this.adapter = adapter;
+	}
+
+	@Override
+	public Stream<Model> apply(Content content) {
+		if (logger.isDebugEnabled()) {
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/AdapterDebugger.java:21]: Is this check necessary? I thought that this logic is already contained in "logger.debug". When you look at AbstractLogger.debug, you see the exact same logic:

`    public void debug(Marker marker, String msg) {
        if (isDebugEnabled(marker)) {
            handle_0ArgsCall(Level.DEBUG, marker, msg, null);
        }
    }`

File: ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/OutputDebugger.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	private final LdiOutput ldiOutput;
+
+	public OutputDebugger(LdiOutput ldiOutput) {
+		this.ldiOutput = ldiOutput;
+		log = LoggerFactory.getLogger(ldiOutput.getClass());
+	}
+
+	@Override
+	public void accept(Model model) {
+		if (log.isDebugEnabled()) {
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/OutputDebugger.java:21]: same question

File: ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/TransformDebugger.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	private final Logger log;
+	private final LdiTransformer ldiTransformer;
+
+	public TransformDebugger(LdiTransformer ldiTransformer) {
+		this.ldiTransformer = ldiTransformer;
+		log = LoggerFactory.getLogger(ldiTransformer.getClass());
+	}
+
+	@Override public Model apply(Model model) {
+		if (log.isDebugEnabled()) {
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/TransformDebugger.java:20]: same question

File: ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/AdapterDebugger.java (hunk 1 above)
Comment (Yalz) [reply ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/AdapterDebugger.java:21]: Sure, will remove

**Pull request merged**